src/HtmlUtils.js

```diff
@@ -208,17 +210,25 @@ module.exports = {
             finally {
                 delete sanitizeHtmlParams.textFilter;
             }
-            return <span className="markdown-body" dangerouslySetInnerHTML={{ __html: safeBody }} />;
         } else {
-            safeBody = content.body;
+            safeBody = _.escape(content.body);
```

Comment:
what's going on with this _.escape()?  I'm dubious about adding a new dependency just for an escaping function, and i'm failing to see why it's needed?